src/installer/pkg/sfx/installers/osx_scripts/host/postinstall

```diff
@@ -10,9 +10,21 @@ INSTALL_DESTINATION=$2
 # A temporary fix for the permissions issue(s)
 chmod 755 $INSTALL_DESTINATION/dotnet
 
-# Add the installation directory to the system-wide paths
-# But first create the directory if it doesn't exist
-mkdir -p /etc/paths.d
-echo $INSTALL_DESTINATION | tee /etc/paths.d/dotnet
+if [ -e /etc/dotnet/install_location ]; then
+    # clear out any entries for this architecture if they exist
+    sed -i old '/^${InstallerTargetArchitecture}=/d' /etc/dotnet/install_location
+else
+    mkdir -p /etc/dotnet    
+fi
+
+echo ${InstallerTargetArchitecture}=$INSTALL_DESTINATION | tee -a /etc/dotnet/install_location
```

Comment:
Side note: For downlevel installers (3.1) we will need to write this without the architecture prefix. The arch prefix is only recognized by 6+ host.

Reply:
Would be nice to put these requirements in the design doc cc @richlander

Reply:
We had a chat about this and will further spec this behavior out.  Will write it up and then have a follow up change if the behavior needs to change here.